src/lib/components/Alert/README.mdx

```diff
@@ -204,6 +204,17 @@ click on the close button:
   }}
 </Playground>
 
+## Forwarding HTML Attributes
+
+In addition to the options below in the [component's API](#api) section, you
+can specify [React synthetic events] or **any HTML attribute you like.** All
+attributes that don't interfere with the API are forwarded to the native HTML
```

Comment:
I dislike the phrase:

> …are forwarded to the native HTML `div`

I think the phrase `native HTML*` makes sense in case of interactive elements that we wrap. In case we transfer the props to the root `div` of some presentation component I would use something along the lines:

> …are forwarded to the root  `div` HTML element

Reply:
I would like to point out it as not always root element (ScrollView, Modal). I am OK with both variants. @adamkudrna?

Reply:
> But this is about Alert, right?

No, this is applicable to many components. most non form components I think.

> I would like to point out it as not always root element

The text has to be altered to reflect how transferProps are handled for the given component. by definition it can not be same for all components.

Reply:
Please, check the last commit. I kept original formulation `native HTML *` for components where `transferProps` are applied to `<button>` or `<input>` HTML element. For remaining components was used the formulation suggested by @mbohal.

Reply:
I like it.